lib/bovine/__init__.py

```diff
@@ -1,5 +1,5 @@
 # from bovine import staticinventory
 from flask import Flask
 
-bovine = Flask(__name__)
-from bovine import api
+app = Flask(__name__)
```

Comment:
actually, now that we renamed api.py to views.py, we are free to instantiate this app for what it is... api. 

so how about
```api = Flask(__name__)```